data/human/events.txt

```diff
@@ -784,6 +784,22 @@ event "initial deployment 4"
 
 
 
+event "fw zug expansion underway"
+	planet Zug
+		spaceport `The spaceport of Zug has been growing by amalgamation for centuries. At its core is the "Old Port," a set of quaint and rudimentary shops and warehouses. The landing pads for the Old Port were torn up to make room for what the locals call "The Port," a set of buildings that were constructed a century ago, with bolder architecture and more modern design.`
+		spaceport `	Some distance away is the "New Port," a towering structure with robotic cargo lifts and large hangars for more expensive ships. Meanwhile, an expansion to the New Port is currently ongoing. You can't help but wonder what the new expansion will be named.`
+
+
+
+event "fw zug expansion complete"
+	planet Zug
+		add shipyard "Tarazed Basics"
+		add shipyard "Tarazed Advanced"
```

Comment:
This feels like it somewhat clashes with #7690, which already adds the Tarazed basics shipyard to Zug. I wonder if this side plot would be better set post-Tarazed assistance event. Then we'd have a much better progression:
* At game start it's just Southbound here.
* Shortly into the first chapter of the campaign, basic Tarazed ships begin to be sold. My PR to #7690 suggests that the ships are still being built at Tarazed but moved into FW systems.
* With this side plot mission, the spaceport could expand into not only selling advanced Tarazed ships, but perhaps building them locally, a big step up from just reselling basic ships.
* Tarazed advanced includes the Falcon, which is bigger than any Southbound ship. Southbound beginning to produce a bigger ship like the Falcon could then lead into them producing the Dreadnought more smoothly.